sdk/search/azure-search-documents/azure/search/documents/__init__.py

```diff
@@ -164,6 +168,10 @@
     "FreshnessScoringParameters",
     "GetIndexStatisticsResult",
     "ImageAnalysisSkill",
+    "Index",
```

Comment:
If you regen'd from swagger, these should be SearchIndex, SearchIndexer, etc. Almost all the renames should actually be right in the newest 2019-05-06-Preview swaggers.